src/PrestaShopBundle/Controller/Admin/TranslationsController.php

```diff
@@ -114,6 +112,7 @@ public function extractThemeAction(Request $request)
         $this->get('prestashop.utils.zip_manager')->createArchive($zipFile, $folderPath);
 
         $response = new BinaryFileResponse($zipFile);
+        $response->deleteFileAfterSend(true);
```

Comment:
You may be surprised by the boolean value, if you pass `false` or nothing the file wont be deleted.

See my proposal here => https://github.com/symfony/symfony/pull/19799